### ISSUE TITLE
Files read in multiple times causing "Duplicated value"

### ISSUE THREAD

**Issue by k-pd** [martin-helmich/typo3-typoscript-lint#56]
I define multiple directories (and multiple file patterns) in tslint.yml. Besides expected warnings and errors I always get many false "Duplicated value" messages. If there are only files in the last directory defined, files are not doubled.

Proof: Create a temp file set with 2 TS files in one directory causing any warnings / errors -> OK, move one file into another directory -> FAIL.
```SH
$ mkdir temp
$ cd temp
temp $ mkdir 1
temp $ mkdir 2
temp $ cat - > tslint.yml
---
paths:
  - 1
  - 2
filePatterns:
  - "*.ts"
^D
temp $ cat - > 2/1.ts
value1=typoscript 1
^D
temp $ cat - > 2/2.ts
value2=typoscript 2
^D
temp $ typoscript-lint 
WW                                                    [2 / 2, 100%]

Completed with 4 issues

CHECKSTYLE REPORT
=> temp/2/1.ts.
   1 No whitespace after object accessor.
   1 No whitespace after operator.
=> temp/2/2.ts.
   1 No whitespace after object accessor.
   1 No whitespace after operator.

SUMMARY
4 issues in total. (4 warnings)
temp $ mv 2/1.ts 1/1.ts
temp $ typoscript-lint 
WWW                                                  [3 / 3, 100%]

Completed with 7 issues

CHECKSTYLE REPORT
=> temp/1/1.ts.
   1 No whitespace after object accessor.
   1 No whitespace after operator.
=> temp/1/1.ts.
   1 No whitespace after object accessor.
   1 No whitespace after operator.
   1 Duplicated value "typoscript". Consider extracting it into a constant.
=> temp/2/2.ts.
   1 No whitespace after object accessor.
   1 No whitespace after operator.

SUMMARY
7 issues in total. (7 warnings)
```

**Comment by mikestreety** [comment]
I am sometimes getting duplicate results too. I am unable to replicate consistently, but will try to narrow down what it is
